README.md

```diff
@@ -1,32 +1,26 @@
 [![Build Status](https://travis-ci.org/mitodl/micromasters.svg?branch=master)](https://travis-ci.org/mitodl/micromasters) [![codecov](https://codecov.io/gh/mitodl/micromasters/branch/master/graph/badge.svg)](https://codecov.io/gh/mitodl/micromasters)
 
 # MicroMasters
-Portal for learners and course teams to access MITx MicroMasters programs
+Portal for learners and course teams to access MITx MicroMasters programs.
 
-## Major Dependencies
-- Docker
-  - [Download from Docker website](https://docker.com/). Follow the OS X or Linux instructions depending on your OS.
-- [Virtualbox](https://www.virtualbox.org/wiki/Downloads) to run edX locally
-- _(OSX only)_ Node/NPM, and Yarn
-  - OSX recommended install method: [Installer on Node website](https://nodejs.org/en/download/)
-  - No specific version has been chosen yet.
+# Initial setup
 
-## (OSX only) Getting your machine Docker-ready
+MicroMasters mostly follows the same [initial setup steps outlined in the common ODL web app guide](https://github.com/mitodl/handbook/blob/master/common-web-app-guide.md).
+Run through those steps in order with the following changes:
 
-#### Create your docker container:
+1. After [creating the database models](https://github.com/mitodl/handbook/blob/master/common-web-app-guide.md#3-create-data-structures), you should create the Elasticsearch indices via management command:
 
-The following commands create a Docker machine named ``default``, start the
-container, and configure environment variables to facilitate communication
-with the edX instance.
+   ```
+   docker-compose run web ./manage.py recreate_index
+   ```
 
-    docker-machine create
-    docker-machine start
-    # 'docker-machine env' prints export commands for important environment variables
-    eval "$(docker-machine env)"
+1. **Do not run the `createsuperuser` command.** When you log into this app via edX, a MicroMasters 
+Django user is created for you automatically. See the section below for instructions on how to run edX locally.
 
-## Running edX devstack locally _(optional, but recommended)_
 
-MicroMasters can work with a live instance of edX, but it's recommended that
+### Running edX devstack locally _(optional, but recommended)_
+
+MicroMasters can work with a deployed instance of edX, but it's recommended that
```

Comment:
I think it would be good here to include information about how to set up MM locally with the edX sandbox, as well as how to run edX locally. I've always used the sandbox and not run into any problems, so for people who aren't working on edX stuff as much it will most likely be an easier option than getting edX running locally.

Reply:
(although if we don't want to add that now that is fine :smile: )

Reply:
Yeah I'd say that's a separate issue. There weren't any details about using the sandbox before this PR. I've never actually used the sandbox. If you have a few minutes for a quick write-up and you think it would be easier, I'd be happy to review and include in this PR

Reply:
eh, it's not so essential to add right now :)